weldx/transformations/types.py

```diff
@@ -13,8 +13,8 @@
 
 types_coordinates = Union[xr.DataArray, np.ndarray, List]
 types_orientation = Union[xr.DataArray, np.ndarray, List[List], Rotation]
-types_time = Union[pd.DatetimeIndex, pd.TimedeltaIndex, pint.Quantity]
-types_time_and_lcs = Union[types_time, "weldx.transformations.LocalCoordinateSystem"]
+types_timeindex = Union[pd.DatetimeIndex, pd.TimedeltaIndex, pint.Quantity]
```

Comment:
how can a quanity be used as an index?

Reply:
array quantites with dimension ["time"] should get converted to `pd.TimedeltaIndex`

Reply:
this means an xarray, right? So a plain quantity is actually misleading here?